keyboards/qwertykeys/qk80/solder/info.json

```
@@ -0,0 +1,112 @@
+{
+    "keyboard_name": "QK80 Solder",
+    "usb": {
+        "pid": "0x3853",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "cols": ["A7", "B0", "B1", "B2", "B10", "B11", "B12", "B13", "B14", "B15", "A8", "B6", "B7", "A6", "A15" ]
+    },
+    "indicators": {
+        "caps_lock": "A0",
+        "on_state": 0
+    },
+    "ws2812": {
+        "pin": "B3"
+    },
+    "layouts": {
+        "LAYOUT_sol": {
```

matches community layout
```suggestion
    },
    "community_layouts": ["tkl_f13_iso_tsangan"],
    "layouts": {
        "LAYOUT_tkl_f13_iso_tsangan": {
```